### PR TITLE
refactor(animations): Remove unnecessary IE specific code

### PR DESCRIPTION
**Pull request by jessicajaniuk**
There were a few places we were still checking for Internet Explorer. This removes the references throughout the animations package.

## PR Checklist
Please check if your PR fulfills the following requirements:

- [x] The commit message follows our guidelines: https://github.com/angular/angular/blob/master/CONTRIBUTING.md#commit
- [ ] Tests for the changes have been added (for bug fixes / features)
- [ ] Docs have been added / updated (for bug fixes / features)

## PR Type
What kind of change does this PR introduce?

- [ ] Bugfix
- [ ] Feature
- [ ] Code style update (formatting, local variables)
- [x] Refactoring (no functional changes, no api changes)
- [ ] Build related changes
- [ ] CI related changes
- [ ] Documentation content changes
- [ ] angular.io application / infrastructure changes
- [ ] Other... Please describe:

## What is the current behavior?

We had a few places where we checked for Internet explorer.

Issue Number: N/A

## What is the new behavior?
We no longer check for internet explorer.

## Does this PR introduce a breaking change?

- [ ] Yes
- [x] No

### CODE REVIEW

File: packages/animations/browser/src/render/shared.ts [hunk 1]
```diff
@@ -173,26 +173,8 @@ if (_isNode || typeof Element !== 'undefined') {
...
-      }
-    } else {
-      const elm = element.querySelector(selector);
-      if (elm) {
-        results.push(elm);
-      }
-    }
-    return results;
+    return multi ? Array.from(element.querySelectorAll(selector)) :
+                   [element.querySelector(selector)];
```
Comment (AndrewKushnir) [inline packages/animations/browser/src/render/shared.ts:177]: This looks like a change in behavior when `querySelector` finds no elements: it used to return an empty array and now it's be an array that contains 1 element (`undefined`).

Comment (jessicajaniuk) [reply packages/animations/browser/src/render/shared.ts:177]: Great catch! I missed that. I'll fix.

**Review by AndrewKushnir — APPROVED** [review]
👍

File: packages/platform-browser/testing/src/browser_util.ts [hunk 2]
```diff
@@ -56,7 +52,7 @@ export class BrowserDetection {
   }
 
   get isSlow(): boolean {
-    return this.isAndroid || this.isIE || this.isIOS7;
+    return this.isAndroid || this.isIOS7;
```
Comment (alfaproject) [inline packages/platform-browser/testing/src/browser_util.ts:55]: IOS 7 is 8 years old or something. Maybe you can consider cleaning up in another pass?

**Comment by atscott** [comment]
This PR was merged into the repository by commit 290df4f806472cb3c3fe1d7cbd132990bf3b15ab.

**Pull request closed without merging**